chrome/content/zotero/xpcom/zotero.js

```diff
@@ -32,6 +32,22 @@ Components.utils.import("resource://gre/modules/PluralForm.jsm");
 
 Services.scriptloader.loadSubScript("resource://zotero/polyfill.js");
 
+var require = (target) => {
+	var { Loader, Require, Module } = Components.utils.import('resource://gre/modules/commonjs/toolkit/loader.js');
+	var requirer = Module('/', '/');
+	var globals = {};
+
+	Components.utils.import("resource://gre/modules/Timer.jsm", globals);
+
+	var loader = Loader({
+		id: 'zotero/requireminimal',
```

Comment:
What's the difference between `zotero/require` above and this? What's the function of this id?